src/styles/global.scss

```diff
@@ -138,6 +140,12 @@ body {
 .catch-phrase{
   font-family: Zilla Slab, san-serif, serif;
   font-style: italic;
-  color: white;
+  color: white
```

Comment:
Consider avoiding the use of named CSS colours. Instead, rely on the colours from your colour palette (i.e., theme).